fairlearn/metrics/_plotter.py

```diff
@@ -0,0 +1,180 @@
+# Copyright (c) Microsoft Corporation and Fairlearn contributors.
```

Comment:
Just Fairlearn contributors will suffice, you can remove Microsoft Corporation from new files.

Reply:
Again, `flake8` should have been warning about that :-/